### PR TITLE
Feature/envs description

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -1,9 +1,8 @@
+##### All variables described in envs.md
 # App configuration
 #############################
 ## Required variables
 
-#### Access to admin panel (route: {app_domain}/admin)
-##### Gen new pass - bin/console security:encode-password 'pass_here' 'Symfony\Component\Security\Core\User\User'
 SUPERADMIN_PASS='$2y$13$BNuHqfNls76Bsqmb8X8UfOTDWbpTwu0VTxPYw65Um0iDw.KPFYzNe'
```
Comment (beshkenadze) [inline .env.dist]: We need to remove it. To be sure that the password will not be used by default.

File: .env.dist [hunk 2]
```diff
@@ -22,54 +21,28 @@ RABBITMQ_DEFAULT_PASS=rabbitmq
...
-##### test - for tests
 APP_ENV=dev
-##### Restriction by domains
 OAUTH_ALLOWED_DOMAINS=example.com,gmail.com
-##### The application needs an immutable random string to add more entropy.
-##### Private signature to encrypt application data (Example: APP_SECRET=af21c49e35f01c1ec4d465daf75098e07a04c7ed)
 APP_SECRET=
-##### Secret hash to encrypt backup codes stored in application data base
-##### (Example: BACKUP_CODE_SALT=af21c49e35f01c1ec4d465daf75098e07a04c7ed)
 BACKUP_CODE_SALT=salt
```
Comment (beshkenadze) [inline .env.dist]: Same here

File: .env.dist [hunk 3]
```diff
@@ -22,54 +21,28 @@ RABBITMQ_DEFAULT_PASS=rabbitmq
...
-##### Secret hash to encrypt invitations stored in application data base
-##### (Example: INVITATION_SALT=af21c49e35f01c1ec4d465daf75098e07a04c7ed)
 INVITATION_SALT=salt
```
Comment (beshkenadze) [inline .env.dist]: And here

File: .env.dist (hunk 1 above)
Comment (mg-4xxi) [reply .env.dist]: As I know removed in separate PR

File: .env.dist (hunk 2 above)
Comment (mg-4xxi) [reply .env.dist]: done

File: .env.dist (hunk 3 above)
Comment (mg-4xxi) [reply .env.dist]: done

**Pull request merged**